acceleration-everest-03-dotnet/AppServices/Validators/UpdateCustomerDtoValidator.cs

```diff
@@ -24,20 +24,20 @@ public UpdateCustomerDtoValidator()
             .Must(customer => customer.EmailConfirmation == customer.Email);
 
         RuleFor(customer => customer.Cpf)
-                .NotEmpty()
-                .MinimumLength(11)
-                .Must(IsValidCpf)
-                .WithMessage("Cpf must be valid.");
+            .NotEmpty()
+            .Length(11)
+            .Must(IsValidCpf)
+            .WithMessage("Cpf must be valid.");
```

Comment:
A sua função IsValidCpf() tem uma verificação de length != 11 return false, é redundante passar o length(11) no validador.
```suggestion
            .NotEmpty()
            .Length(11)
            .Must(IsValidCpf)
            .WithMessage("Cpf must be valid.");
```
```suggestion
            .NotEmpty()
            .Must(IsValidCpf)
            .WithMessage("Cpf must be valid.");
```